docs/lang/articles/basic/syntax.md

```diff
@@ -36,16 +34,16 @@ def my_kernel():
 my_kernel()
 ```
 
-Kernels should be called from **Python-scope**.
+Kernels should be called from **Python-scope**, nested kernels are not supported.
 
 :::note
-For people from CUDA, Taichi kernels = `__global__` functions.
+For people from CUDA, Taichi kernels are similar to `__global__` functions.
 :::
 
 ### Arguments
 
-Kernels can have at most 8 parameters so that you can pass values from
-Python-scope to Taichi-scope easily.
+Kernels can have multiple arguments (For kernels executed on OpenGL and cc backends, the number of arguments is limited to 8.), which support passing values from
```

Comment:
Can we make *For kernels executed on OpenGL and cc backends, the number of arguments is limited to 8.* a note? I.e.,
```
:::note
For kernels executed on OpenGL and cc backends, the number of arguments is limited to 8.
:::note
```

Reply:
Nice suggestions! Have made it a note.